docs/website/docs/building-from-source/ios.md

```diff
@@ -0,0 +1,130 @@
+# Cross-compilation for iOS
+
+Cross-compilation for iOS consists of the two steps below.
+
+* On the macOS host, build the IREE compiler.  We can run it to create
+  IREE modules.
+* Build the IREE runtime on the macOS host for iOS devices and the
+  simulator.  We can then run the IREE module on the simulator.
```

Comment:
We can consider flipping the check marks in https://iree-org.github.io/iree/#support-matrix for macOS/iOS either as part of this change or in the future.

The source for that is here: https://github.com/iree-org/iree/blob/6907d954b11937dc767344d799b0e563080aed08/docs/website/docs/index.md?plain=1#L33-L39

Reply:
Great idea! I added a paragraph about running the generated IREE module on the macOS host and checked "macOS" as well.